docs/en/setup/service-agent/java-agent/TLS.md

```diff
@@ -17,10 +17,23 @@ Only support **no mutual auth**.
 ## Open and config TLS
 
 ### Agent config
-- Place `ca.crt` into `/ca` folder in agent package. Notice, `/ca` is not created in distribution, please create it by yourself.
```

Comment:
There was a sentence `Only support no mutual auth.` in this doc, please fix.

Reply:
Specifically, line 11 @dmsolr